File: src/main/java/com/tom/Hello.java
package com.tom;

import java.util.Scanner;

public class Hello {
    public static void main(String[] args) {
//        System.out.println("Hello world");
        String s = "abcde";
        System.out.println(s.charAt(3));
        System.out.println(s.substring(1,4));
        Person p = new Person("Tom", 66.5f, 1.7f);
        p.hello();
        p.weight = 66.5f;
        p.height = 1.7f;
        System.out.println(p.bmi());
        int score = 88;
        System.out.println(score < 80 || score > 90);
        char c = 'A';
        System.out.println(c > 'a');
        System.out.println("Please enter a number");
        Scanner scanner = new Scanner(System.in);
        int n = scanner.nextInt();
        System.out.println(n > 60);
        /*int age = 19;
        Integer age2 = 19;
        char c = '我';
        Character c2 = 'A';
        byte b = 120;
        float weight = 66.5f;
        float height = 1.7f;
        boolean adult = true;
        boolean enroll = false;
        String name = "Tom";*/
    }
}

File: src/main/kotlin/com/tom/Hello.kt
package com.tom

fun main(args: Array<String>) {
    var s:String? = "abcde"
    s = null
    println(s?.length)
    println(s?.get(3))
    println(s?.substring(3))
//    println("Hello kotlin")
    val h = Human(weight = 66.5f, height = 1.7f)
    println(h.bmi())
    val score = 88
    println(score > 60)
    val c :Char = 'A'
    println(c.toInt() > 60)
    /*var age = 19
    age = 20
    var weight = 66.5f
    var name : String
    name = "Hank"*/

}

class Human(var name: String = "", var weight: Float, var height:Float) {
    init {
        println("test $weight" )
    }
//    constructor(name:String, weight: Float, height: Float) :this(weight, height)
    fun bmi() :Float {
        val bmi = weight / (height*height)
        return bmi
    }

    fun hello() {
        println("Hello kotlin")
    }
}